pkg/label/label.go

```diff
@@ -6,12 +6,22 @@
 
 package label
 
+import (
+	snpkg "github.com/containerd/containerd/pkg/snapshotters"
+)
+
+// For package compatibility, we still keep the old exported name here.
+var AppendLabelsHandlerWrapper = snpkg.AppendInfoHandlerWrapper
+
+// For package compatibility, we still keep the old exported name here.
```

Comment:
This compatibility trick looks good to me.  Hopefully, we can update this package's consumer to let them directly import `containerd/snapshotters` packge.